syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -58,9 +58,6 @@ class TorchTensor(AbstractTensor):
     origin = None
     id_at_origin = None
 
-    def has_child(self):
```

Comment:
Q: Isn't this ```has_child``` method specific to only tensors?
I am thinking maybe it is worth putting it in the ```abstract/tensor.py```?
@LaRiffle

Reply:
Yeah that's right! @marload maybe let's move it there and leave syft/generic/abstract/object.py  untouched?

Reply:
I applied your suggestion!